tests/data.py

```diff
@@ -2,14 +2,20 @@
 
 
 class Actor(Enum):
-    # User ID, org code, business function, OBO User ID
-    RC = ("555031999105", "D82106", "REFERRING_CLINICIAN")
-    RCA = ("555031998104", "D82106", "REFERRING_CLINICIAN_ADMIN")
-    RA = ("555031998103", "D82106", "REFERRING_ADMIN")
-    SPC = ("555032006106", "RCD", "SERVICE_PROVIDER_CLINICIAN")
-    SPA = ("555032006103", "RCD", "SERVICE_PROVIDER_ADMIN")
-    SPCA = ("555031998104", "RCD", "SERVICE_PROVIDER_CLINICIAN_ADMIN", "555031999105")
-    RC_DEV = ("021600556514", "R69", "REFERRING_CLINICIAN")
+    # User ID, org code, business function, IAL, OBO User ID
+    RC = ("555031999105", "D82106", "REFERRING_CLINICIAN", "2")
```

Comment:
Yes.

Reply:
Did you get to the bottom of why other tests utilising this user don't seem to be failing?

Reply:
Yes, I think so. As discussed, this is because the user is only used on existing tests in non-mocked environments. I have applied a fix whereby a different user with IAL 3 is used instead, for this purpose. I am just awaiting ALPHA to be started, to test this fix ([ERSSUP-76214](https://nhsd-jira.digital.nhs.uk/browse/ERSSUP-76214)).